<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cars_db</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="input_field"/>
      </item>
      <item>
       <widget class="QPushButton" name="search_button">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputID">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777176</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Brand:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputBrand"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Model:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputModel"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputYear"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="colorbox">
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>purple</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addbutton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="output_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>search_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchcar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addcar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>input_field</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputYear</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputModel</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputBrand</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addbutton</sender>
   <signal>clicked(bool)</signal>
   <receiver>inputID</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchcar()</slot>
  <slot>addcar()</slot>
 </slots>
</ui>
